docs/reference/metadata.md

```diff
@@ -175,37 +175,77 @@ Indentation options control aspects of the auto-indentation mechanism.
 `increaseIndentPattern`
 : *Regex.*
   If it matches on the current line,
-  the next line will be indented one level further.
+  lines after current line will be indented one level further.
+
+  If in `.tmPreferences`:
 
   ```xml
   <key>increaseIndentPattern</key>
-  <string>insert regex here</string>
+  <string><![CDATA[^\s*increase$]]></string>
+  ```
+
+  Then in `view`:
+
+  ```
+  line1
+  line2
+  increase
+    after increase1
+    after increase2
   ```
 
 `decreaseIndentPattern`
 : *Regex.*
   If it matches on the current line,
-  the next line will be unindented one level.
+  current line and lines after current line will be unindented one level.
+
+  If in `.tmPreferences`:
 
   ```xml
   <key>decreaseIndentPattern</key>
-  <string>insert regex here</string>
+  <string><![CDATA[^\s*decrease$]]></string>
+  ```
+
+  Then in `view`:
+
+  ```
+    line1
+    line2
+  decrease
+  after decrease1
+  after decrease2
   ```
 
 `bracketIndentNextLinePattern`
 : *Regex.*
   If it matches on the current line,
-  only the next line will be indented one level further.
+  only the next non-whitespace line will be indented one level further.
+
+  If in `.tmPreferences`:
 
   ```xml
   <key>bracketIndentNextLinePattern</key>
-  <string>insert regex here</string>
+  <string><![CDATA[^\s*bracket$]]></string>
+  ```
+
+  Then in `view`:
```

Comment:
I'm not convinced "view" needs to be backtick-quoted here. It might be better to say something like "Then, in a buffer for which this tmPreferences file is active", though I agree that is a bit of a verbose mouthful.